helm/ngrok-operator/values.yaml

```diff
@@ -204,6 +204,12 @@ serviceAccount:
   name: ""
   annotations: {}
 
+## Cluster Role Settings
+## @param clusterRole.annotations Additional annotations to add to all ClusterRoles
```

Comment:
It might be worth putting a little explanation here on what you would use these annotations for. k8s has some decent docs here we could link to:
- https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles
- https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles

This might be worth working with @iambmelt on some doc updates as well as a follow up. I could see us having a doc page on hardening the security of the operator/cluster and documenting how to restrict CRD access there.